src/server/transaction/pretty/pretty.go

```diff
@@ -65,9 +65,9 @@ func sprintDeleteRepo(request *pfs.DeleteRepoRequest) string {
 		force = " --force"
 	}
 	if request.All {
-		return fmt.Sprintf("delete repo --all%s", request.Repo.Name, force)
```

Comment:
oh uh, I came across this as well in my upcoming PR - I think we still want the `force` here.

Reply:
Oh good call, I'll add the `force` back in.